### ISSUE TITLE
[Bug]: Shader's not's working

### ISSUE THREAD

**Issue by otavioxw** [AllTheMods/ATM-10#1039]
### Possible Fixes

Yes

### Modpack Version

ATM10-1.16

### What happened?

My shaders stopped working in version 1.16; in the previous version (1.15), the enable/disable keys worked and did the same, but in 1.16, I tried disabling and re-enabling to see if the process was working, and the shaders didn't work again

### latest.log

https://gist.github.com/otavioxw/9bb36765f13517ed483d8cdc08964f21

**Comment by MollyXVI** [comment]
i also have shader problems, often when entering ro leaving a dimention, the shaders crash and stop working and i have to switch to another before that too fail.

**Comment by HiddenKacper** [comment]
Same issue here, shaders turned off by itself and cant be turned on, changing shaders pack causes crash, when it happend i was in overworld doing some basic AE2 things.

**Comment by Mohawk1025** [comment]
The issue is with Mekanism Covers. The Iris developer confirmed. I just checked ATM 10 1.17 and the issue still persists.

https://github.com/IrisShaders/Iris/issues/2517

**Comment by MollyXVI** [comment]
My issue also still persists in 1.17, however i dont use or have any covers. Im using Mekanism Logistical Transporters however and the items moving inside are glitching and freaking out.

**Comment by Mohawk1025** [comment]
> My issue also still persists in 1.17, however i dont use or have any covers. Im using Mekanism Logistical Transporters however and the items moving inside are glitching and freaking out.

I don't think it matters if you're using the covers or not. It's still in the modpack, and the developer confirmed there was an issue. I've stayed on 1.15 until it's fixed, because I like having shaders.

**Comment by 19Arice** [comment]
You can still have shaders if you update, just  change the   "disableAdvancedCoverRendering": true in \config\mekanismcovers.json in your forge client save.

**Comment by Mohawk1025** [comment]
> You can still have shaders if you update, just change the "disableAdvancedCoverRendering": true in \config\mekanismcovers.json in your forge client save.

Yeah, in another thread on this issue someone mentioned that. Also, in the latest version of ATM 10 they made this change themselves, so everyone who updates should see this issue resolved.
